principles.md

```diff
@@ -0,0 +1,20 @@
+## Product and Engineering Principles
+
+**We believe in being paid for quality work**
+
+There's an incredible amount of software infrastructure that is under funded and ignored.
+We believe our software needs to generate revenue to employ the best minds.
+
+**We balance openness and transparency**
```

Comment:
```suggestion
**We value openness and transparency**
```

Reply:
My thought on balance was that this is more pragmatic to how the company actually acts, compare to something like the entire product being GPL and opensource not open-core.

Reply:
OK, but `balance openness and transparency` reads as `balance between openness and transparency`.
These are synonyms, so it was confusing to me :)